ansible/tasks/logging/README.md

```diff
@@ -0,0 +1,34 @@
+# Playbooks to install FluentD and FluentD plugins
+
+These playbooks install FluentD on Windows instance in order to enable logging support.
+
+Before running these playbooks,
+- Set up an OpenShift cluster
+- Ensure Ansible environment is set up to run the playbook. For a Fedora user, sample steps are:
+```
+$ sudo dnf install libselinux-python
+$ pip install ansible pywinrm
+```
+- A Windows Server 2019 running in the same VPC as the OpenShift cluster with winrm enabled.
+You can use `wni` tool to setup the Windows node following this [link](https://github.com/openshift/windows-machine-config-operator/tree/master/tools/windows-node-installer).
+The winrm module is enabled automatically if you use `wni` tool to create the server node.
+- Login to your cluster using `oc` command.
```

Comment:
Why should we login? Are we using any subsequent `oc` commands in the ansible playbook?

Reply:
Logging into the cluster is not required for this playbook